geocoding.py

```diff
@@ -1,25 +1,60 @@
+'''Geocoding Flask project
+This program is a flask project that allows a user to input an address
+and return to the user the latitude and longitude of that address.
+
+The script requires that requests be installed as well as flask within
+the python enviroment you are running the program in.
+'''
 import requests
+from flask import Flask, render_template, request
+
+app = Flask(__name__)
 
 # Base Url for geocoding
-url = "https://us1.locationiq.com/v1/search.php"
 
-address = input("Input the address: ")
+def get_geo(address):
+    '''
+    Function that holds all of the functionality to find latitude
+    and longitude
+    '''
+    url = "https://us1.locationiq.com/v1/search.php"
+    private_token = "pk.727f7e39de9a6cafee73b56668557864"
```

Comment:
I would still make url and private token global up near line 11 where app is initialized . Otherwise those variables are going to be intialized every time that function is called which isn't necessary. They only need to be initialized once. 

```
URL = .. 
PRIVATE_TOKEN = ...    
```